include/vcpkg/base/messages.h

```diff
@@ -565,6 +570,11 @@ namespace vcpkg
                     (msg::count, msg::vendor),
                     "",
                     "Attempting to fetch {count} package(s) from {vendor}");
+    DECLARE_MESSAGE(AttemptingToSetBuiltInBaseline,
+                    (),
+                    "",
+                    "attempting to set builtin-baseline in vcpkg.json while overriding the default-registry in "
+                    "vcpkg-configuration.json.\n\tthe default-registry from vcpkg-configuration.json will be used.");
```

Comment:
```suggestion
                    "vcpkg-configuration.json.\nthe default-registry from vcpkg-configuration.json will be used.");
```

I know the old code used a tab here but that's inconsistent with how we do identing elsewhere, and I'm not seeing a strong reason to split this into separate messages to keep the indenting.